traveler_ui_kit/src/main/java/com/guestlogix/traveleruikit/fragments/ActionStripContainerFragment.java

```diff
@@ -18,13 +18,13 @@
  * Fragment to contain Action strip based on Purchased Strategy
  */
 public class ActionStripContainerFragment extends Fragment {
-    public static final String ARG_ITEM = "ARG_ITEM";
+    public static final String ARG_Product = "ARG_Product";
```

Comment:
this work is also outside the commit's scope. IF you're going to do it, then add it to the commit message.